Readability.js

```diff
@@ -721,11 +721,11 @@ Readability.prototype = {
           }
         }
 
-        // Remove empty DIV, SECTION, and HEADER nodes
+        // Remove DIV, SECTION, and HEADER nodes without any content(e.g. text, image, video, or iframe).
```

Comment:
I'm still wary that we have a list of elements that avoid removal, rather than a list of elements that don't prevent removal. So if we're forgetting about non-text elements that aren't images or video/audio/iframe elements, that *should* trigger keeping the container, this code is buggy. I would prefer it if it treated elements being either empty or containing only br/hr elements as superfluous, rather than the wider approach this patch is taking. Does that make sense?